en/02_Developer_Guides/10_Email/index.md

```diff
@@ -193,16 +183,14 @@ $this->send();
 You can set the default sender address of emails through the `Email.admin_email` [configuration setting](/developer_guides/configuration).
 
 **app/_config/app.yml**
-
-
-```yaml
+```yml
```

Comment:
Do you know for a fact this will format correctly? The existing `yaml` is definitely formatting correctly so I'm hesitant to change this. Probably best to leave these alone for now - there's [an existing card](https://github.com/silverstripe/developer-docs/issues/11) to standardise this sort of thing so if we want to standardise `yaml` to `yml` we can do it as part of that card.
I've added standardisation of `yaml` or `yml` to the card's proposed changes.

Reply:
Reverted